app/Main.hs

```diff
@@ -262,6 +265,12 @@ optsParser =
       [ metavar "FILE",
         help "Haskell source files to format or stdin (the default)"
       ]
+    <*> (fmap (fromMaybe 1) . optional . option auto . mconcat)
```

Comment:
I think it would be a bit more idiomatic to use [`value`](https://hackage.haskell.org/package/optparse-applicative-0.17.0.0/docs/Options-Applicative.html#v:value) and [`showDefault`](https://hackage.haskell.org/package/optparse-applicative-0.17.0.0/docs/Options-Applicative.html#v:showDefault) instead of `fmap (fromMaybe 1) . optional`.